lib/broker.coffee

```diff
@@ -37,6 +41,12 @@ module.exports = class Broker extends EventEmitterWildcard
     @subscriptionThrottleMs = 2000
     @unsubscriptionThreshold = 40
 
+    @readyState = READY
+    @emit 'ready'
+    @emit 'connected'
+    trace "fake init"
+
+    return
     @initBackoff options.backoff  if @autoReconnect
     @connect()
```

Comment:
should we put some comments here why we did this and instead of keeping the code after `return` statement we can remove them I believe.